app/models/edition/specialist_sectors.rb

```diff
@@ -48,6 +48,14 @@ def specialist_sector_tags
     specialist_sectors.order("specialist_sectors.primary DESC").map(&:topic_content_id)
   end
 
+  def has_primary_sector?
+    !primary_specialist_sector_tag.blank?
+  end
+
+  def has_secondary_sectors?
+    secondary_specialist_sector_tags.any?
+  end
```

Comment:
You could implement `has_specialist_sectors?`, as `specialist_sectors.any?` instead maybe?

Reply:
I've seen that both methods are used in the view, so maybe this shouldn't be consolidated here...